packages/components/input-number/src/index.vue

```diff
@@ -249,6 +249,15 @@ export default defineComponent({
       }
       data.userInput = null
     }
+
+    const focus = () => {
+      input.value.focus()
+    }
+
+    const blur = () => {
+      input.value.blur()
+    }
```

Comment:
Would be better with optional chaining though.
E.g.

```ts
input.value.blur?.()
```

Reply:
This could help us with edge case

Reply:
@JeremyWuuuuu  Yeah, that's a good idea.